src/firefly/js/visualize/ChartsTableViewPanel.jsx

```diff
@@ -357,13 +362,13 @@ var ChartsPanel = React.createClass({
                          src={ZOOM_IN}
                          onClick={() => this.addZoom()}
                     />
-                    <img style={selectionBtnStyle}
-                         title='Select enclosed points - ToDo'
+                    {!get(this.props, 'tblPlotData.xyPlotData.decimateKey') && <img style={selectionBtnStyle}
+                         title='Select enclosed points'
                          src={SELECT_ROWS}
                          onClick={() => this.addSelection()}
-                    />
+                    />}
                     <img style={selectionBtnStyle}
-                         title='Filter in the selected points - ToDo'
+                         title='Filter in the selected points'
                          src={FILTER_IN}
                          onClick={() => this.addFilter()}
```

Comment:
how about change the cursor while hovering the img in order to give the user a hint to click the img?